### DISCUSSION TITLE
The addon skill change alone enabling elvui

### DISCUSSION THREAD

**Discussion by doctorase** [SavedInstances/SavedInstances#906] [Q&A] [answered]
Hi. When I use elvui the addon changes the skin and I can't change the size of the window or anything. I have asked in the Elvui discord and they have told me that they don't have any code related to this addon, so the problem must come with this addon itself. Only by disabling and enabling elvui the SavedInstance skin changes by itself. 

Any way to keep the normal skin with elvui enabled?

**Comment by LiangYuxuan** [comment, marked as answer]
No, SI has build-in skin function using ElvUI or Tukui, this is not able to disable. Skinning isn't disabling any function, like resizing which is not possible.
